examples/tests/valid/bring_jsii_path.w

```diff
@@ -0,0 +1,9 @@
+bring cloud;
+bring "./node_modules/jsii-code-samples" as jsii_code_samples;
```

Comment:
Why do we need `./node_modules` here? Shouldn't this behave like `require`?

Reply:
We don't need it, but this file is testing the behavior of importing from a path rather than a node module (https://github.com/winglang/wing/issues/1488)

Reply:
there is already another test that does `bring "jsii-code-samples"`